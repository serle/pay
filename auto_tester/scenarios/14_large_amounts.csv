type,client,tx,amount
deposit,1,1,999999.9999
withdrawal,1,2,500000.5000
deposit,1,3,0.0001
