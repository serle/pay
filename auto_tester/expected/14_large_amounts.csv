client,available,held,total,locked
1,499999.5000,0.0000,499999.5000,false
